src/fault_injector/fault_types/platform_blocked_fault.py

```diff
@@ -35,10 +41,8 @@ def resolve_fault(self, tick: int):
         :param tick: the simulation tick in which resolve_fault was called
         :type tick: Integer
         """
-        if self.platform is None:
-            raise ValueError(
-                "Platform not set, probably due to not injecting the fault"
-            )
+        if self.platform is None or self.platform is not self._get_platform():
+            raise ValueError("Platform does not exist or fault not injected")
```

Comment:
```suggestion
        if self.platform is None: or self.platform is not self._get_platform():
            raise ValueError("Fault was not injected")
        if  self.platform is not self._get_platform():
            raise ValueError("Platform does not exist.")
```
This change could make it more descriptive.